src/main/java/com/turtywurty/railroad/RailroadIDE.java

```diff
@@ -69,53 +73,54 @@ public void createComponents(Stage window) {
+	// TODO: Make proper textures for all these icons. Currently all just programmer
+	// art! ;)
 	public Menu createNewMenu() {
-		Menu menu = new Menu("New");
+		Menu menu = new Menu(this.config.lang.get("menu.file.new"));
 
-		MenuItem javaProject = RailroadMenuItem.Builder.create("Java Project")
+		MenuItem javaProject = RailroadMenuItem.Builder.create(this.config.lang.get("menu.file.new.javaproject"))
 				.setGraphic(Utils.createMenuGraphics("/assets/img/java_project.png")).build();
 
-		MenuItem project = RailroadMenuItem.Builder.create("Project")
+		MenuItem project = RailroadMenuItem.Builder.create(this.config.lang.get("menu.file.new.project"))
 				.setGraphic(Utils.createMenuGraphics("/assets/img/project.png")).build();
 
 		// TODO: Once added texture at "/assets/img/working_set.png" un-comment
 		// setGraphic
-		MenuItem javaWorkingSet = RailroadMenuItem.Builder.create("Java Working Set")
+		MenuItem javaWorkingSet = RailroadMenuItem.Builder.create(this.config.lang.get("menu.file.new.javaworkingset"))
 				/* .setGraphic(Utils.createMenuGraphics("/assets/img/working_set.png") */.build();
 
-		MenuItem packagE = RailroadMenuItem.Builder.create("Package")
+		MenuItem packagE = RailroadMenuItem.Builder.create(this.config.lang.get("menu.file.new.package"))
 				.setGraphic(Utils.createMenuGraphics("/assets/img/package.png")).build();
 
-		MenuItem sourceFolder = RailroadMenuItem.Builder.create("Source Folder")
+		MenuItem sourceFolder = RailroadMenuItem.Builder.create(this.config.lang.get("menu.file.new.sourcefolder"))
 				.setGraphic(Utils.createMenuGraphics("/assets/img/source_folder.png")).build();
 
-		MenuItem file = RailroadMenuItem.Builder.create("File").setGraphic(Utils.createMenuGraphics("/assets/img/file.png"))
-				.build();
+		MenuItem file = RailroadMenuItem.Builder.create(this.config.lang.get("menu.file.new.file"))
+				.setGraphic(Utils.createMenuGraphics("/assets/img/file.png")).build();
 
-		MenuItem folder = RailroadMenuItem.Builder.create("Folder")
+		MenuItem folder = RailroadMenuItem.Builder.create(this.config.lang.get("menu.file.new.folder"))
 				.setGraphic(Utils.createMenuGraphics("/assets/img/folder.png")).build();
 
-		MenuItem clazz = RailroadMenuItem.Builder.create("Class")
+		MenuItem clazz = RailroadMenuItem.Builder.create(this.config.lang.get("menu.file.new.class"))
 				.setGraphic(Utils.createMenuGraphics("/assets/img/class.png")).build();
 
-		MenuItem interfacE = RailroadMenuItem.Builder.create("Interface")
+		MenuItem interfacE = RailroadMenuItem.Builder.create(this.config.lang.get("menu.file.new.interface"))
 				.setGraphic(Utils.createMenuGraphics("/assets/img/interface.png")).build();
 
-		MenuItem enuM = RailroadMenuItem.Builder.create("Enum").setGraphic(Utils.createMenuGraphics("/assets/img/enum.png"))
-				.build();
+		MenuItem enuM = RailroadMenuItem.Builder.create(this.config.lang.get("menu.file.new.enum"))
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Use of identifier that is a keyword in later versions of Java (NM_FUTURE_KEYWORD_USED_AS_IDENTIFIER)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-use-of-identifier-that-is-a-keyword-in-later-versions-of-java-nm-future-keyword-used-as-identifier).
The identifier is a word that is reserved as a keyword in later versions of Java, and your code will need to be changed in order to compile it in later versions of Java.